CONTRIBUTING.md

```diff
@@ -13,13 +13,13 @@ Crayons are built using StencilJS which provides all features of web component.
 
 ## How to Contribute
 
-Crayons have a predefined set of components, As a developer, you can not only use the components in your projects, you can also contribute to Crayons as it falls under MIT license and we welcome all contributions, small or big.
+Crayons has a predefined set of components. As a developer, you not only can use the components in your projects but you can also contribute to Crayons, as it falls under a MIT license. We welcome all contributions, big or small.
```

Comment:
Could we modify this as follows?

Crayons has a predefined set of components. As a developer, you not only can use the components in your projects but can also contribute to Crayons, as it falls under the MIT license. We welcome all contributions, big or small.